tests/acceptance/smoke-test-slow.js

```diff
@@ -82,6 +82,19 @@ describe('Acceptance: smoke-test', function() {
       });
   });
 
+  it('ember test exits with zero when tests pass', function() {
+    return copyFixtureFiles('smoke-tests/passing-test')
+      .then(function() {
+        return runCommand(path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember'), 'test')
```

Comment:
@seanpdoyle example here https://github.com/seanpdoyle/ember-cli/blob/sd-module-for-acceptance-smoke-test/tests/acceptance/smoke-test-slow.js#L172, a faster way to run ember-cli in tests.